lib/nexus/v1/component.js

```diff
@@ -11,29 +11,35 @@ class Component {
    * @constructor
    */
   constructor(data) {
+
     /**
      * Name of the component, display value
      * @type {string}
      */
     this.name = data.name;
 
     /**
-     * Average price of the component as a string
-     * @type {string}
+     * Supply for the component
+     * @type {Object.<number>}
      */
-    this.avgPrice = data.avg;
+    this.supply = data.supply
 
     /**
-     * Raw average price of the component as a number
-     * @type {number}
+     * Demand for the component
+     * @type {Object.<number>}
```

Comment:
Don't need the `Object.<number>` specification. if it's either an object or number, it'd be `Object | number`